doc/api/url.md

```diff
@@ -251,15 +251,15 @@ const myURL = new URL('https://example.org:81/foo');
 console.log(myURL.hostname);
 // Prints example.org
 
-// Setting the hostname does not change the port
+// Setting an invalid hostname is ignored:
```

Comment:
```suggestion
// Setting the hostname does not change the port
```

The sentence on line 265 already clarifies that invalid host name values are ignored.